pcsx2-qt/MainWindow.cpp

```diff
@@ -2381,6 +2382,20 @@ void MainWindow::doSettings(const char* category /* = nullptr */)
 		dlg->setCategory(category);
 }
 
+DebuggerWindow* MainWindow::getDebuggerWindow()
+{
+	if (!m_debugger_window)
```

Comment:
Did you want to close the debugger window when the VM is stopped? (to prevent it from messing with nonexistant state). If so, you can clean it up in `onVMStopped()`.

Reply:
I don't mind leaving it open, sometimes my workflow requires testing multiple ELFs, stopping the VM in-between and stuff.